mat/triangular.go

```diff
@@ -105,6 +105,7 @@ func (t TransposeTri) Untranspose() Matrix {
 	return t.Triangular
 }
 
+// UntransposeTri and return specifically a Triangular object.
```

Comment:
These documentation changes are incidental and should be in a separate PR.

Reply:
ive just removed them.